src/Compilers/CSharp/Portable/FlowAnalysis/UnassignedFieldsWalker.cs

```diff
@@ -113,7 +110,7 @@ private bool GetIsSymbolAssigned(int thisSlot, Symbol symbol)
             {
                 return true;
             }
-            int slot = VariableSlot(symbol, thisSlot);
+            int slot = VariableSlot(symbol, symbol.IsStatic ? 0 : thisSlot);
```

Comment:
>IsStatic [](start = 51, length = 8)

Should this be `symbol.RequiresInstanceReceiver()` as in LocalDataFlowPass?

Reply:
Yes, I think that's correct.